### PR TITLE
chore: add cloudwatch alarms

### LINKED ISSUE #75: CloudWatch Alarms
Set up a few basic CloudWatch Alarms on a couple of critical metrics (for instance Lambda failures) I suggest also creating an SNS topic to trigger with those alerts so that the maintainers who have access to the PROD aws account can subscribe to it.

### PR DESCRIPTION
**Pull request by wperron**
* add an alarm for total execution errors over 10 minutes for each
  lambda function
* add an alarm for old messages in the build queue
* add an alarm for total visible messages in the dead letter queue
* parameterized the aws region

Make sure to create at least one topic subscription after rolling out these changes to get notified by alerts.

Fixes #75

### CODE REVIEW

File: terraform/alerts.tf [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  evaluation_periods  = 1
+  period              = 300
+  datapoints_to_alarm = 1
+  statistic           = "Average"
+  metric_name         = "ApproximateNumberOfMessagesVisible"
+  namespace           = "AWS/SQS"
+  dimensions = {
+    "QueueName" = aws_sqs_queue.build_dlq.name
+  }
+  threshold          = 25
```
Comment (lucacasonato) [inline terraform/alerts.tf:62]: Does this mean 25 messages need to be in the DLQ for the alarm to go off?

**Review by lucacasonato — COMMENTED** [review]
Very nice

File: terraform/alerts.tf (hunk 1 above)
Comment (wperron) [reply terraform/alerts.tf:62]: Yup (plus or minus some, SQS can't give an exact number)

Comment (lucacasonato) [reply terraform/alerts.tf:62]: :+1: - I don't expect this alert to ever trigger anyway because the max age alert will likely kick in first.

Comment (wperron) [reply terraform/alerts.tf:62]: yup, it's really just to cover our bases

**Review by lucacasonato — COMMENTED** [review]
Could you add an `alarm_description` to each alarm too?

File: terraform/variables.tf [hunk 2]
```diff
@@ -1,3 +1,8 @@
+variable "region" {
```
Comment (lucacasonato) [inline terraform/variables.tf:1]: Can you add this to the `terraform.tfvars.example` file?

Comment (wperron) [reply terraform/variables.tf:1]: done!

**Review by lucacasonato — APPROVED** [review]
Thanks!

**Pull request merged**